numpy/lib/_function_base_impl.py

```diff
@@ -388,7 +388,7 @@ def iterable(y):
 
 def _weights_are_valid(weights, a, axis):
     """Validate weights array.
-    
+
```

Comment:
I suspect these are changes from an autoformatter. Some projects prefer not to change existing code, even when not complying with the formatting conventions to avoid churn. Not sure what the numpy policy is though